experimental/traffic_ops_auth/traffic_ops_auth.go

```diff
@@ -192,7 +192,7 @@ func HandleLogin(db *sql.DB, jwtSigningKey string, w http.ResponseWriter, r *htt
 	clms, ok := token.Claims.(jwt.MapClaims)
 	if !ok {
 		w.WriteHeader(http.StatusInternalServerError)
-		log.Printf("%s ERROR creating token for '%s': expected MapClaims in token, got %T", token.Claims)
+		log.Printf("ERROR creating token for '%s': expected MapClaims in token, got %T", "token.Claims", token.Claims)
```

Comment:
Why bother using a format arg for an inline literal?

Reply:
Why indeed. Inlined the string literal in 5f2d70dc26.